ui-v2/app/routes/dc/nspaces/edit.js

```diff
@@ -0,0 +1,35 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { hash } from 'rsvp';
+
+import WithNspaceActions from 'consul-ui/mixins/nspace/with-actions';
+
+export default Route.extend(WithNspaceActions, {
+  repo: service('repository/nspace'),
+  isCreate: function(params, transition) {
+    return transition.targetName.split('.').pop() === 'create';
```

Comment:
How does create end up on the edit route?

Reply:
The create route extends the edit route, so they are both the same, but occasionally we need to know if its one or the other.